### DISCUSSION TITLE
ROUGE mesure

### DISCUSSION THREAD

**Discussion by Maria2244** [huggingface/datasets#5676] [General]
hi 
i have question 
i have N Summery with their predict Summery 
how i get the ROUGE measure can some one help me please
. i used this method 
first i add all summery in list 1 and all predict summery in list 2 then i implement the ROUGE is it correct?
please some one guide me ?

**Comment by mariosasko** [comment]
Hi! The metrics in `datasets` are deprecated and will be removed in the next major release of `datasets`, so please use `evaluate` instead for metrics. You can also find an [example](https://github.com/huggingface/evaluate/tree/main/metrics/rouge) of how to use the ROUGE metric there.

**Reply by Maria2244** [reply]
thank you
i used this but get error 
------------------------------------------
from rouge_score import rouge
import evaluate
rouge_scorer = evaluate.load('rouge')
predictions = [h]
references = [r]
results = rouge.compute(predictions=predictions,references=references)
print(results)
ModuleNotFoundError: No module named 'evaluate'

**Reply by mariosasko** [reply]
You need to install the `evaluate` package before running the code with `pip install evaluate`
